plan/logical_plan_builder.go

```diff
@@ -1168,3 +1204,19 @@ func (b *planBuilder) buildDelete(delete *ast.DeleteStmt) LogicalPlan {
 	del.SetSchema(expression.NewSchema())
 	return del
 }
+
+func extractTableListFromPlan(p LogicalPlan) []*DataSource {
+	var ret []*DataSource
+	for p != nil {
+		switch v := p.(type) {
```

Comment:
I guess `delete from a,b,c` generate a list, chained by join's left child , is that true?